docs/topics/optimize.rst

```diff
@@ -0,0 +1,164 @@
+.. _optimize:
+
+============
+Optimization
+============
+
+Scrapy offers different ways to speed up crawls for specific :ref:`use cases
+<broad-crawls>` and to :ref:`lower resource usage <optimize-resources>`.
+
+.. _broad-crawls:
+.. _topics-broad-crawls:
+
+Speeding up broad crawls
+========================
+
+While Scrapy is well suited for **broad crawls**, i.e. crawls that target many
+websites, the default :ref:`settings <topics-settings>` are optimized for
+crawls targetting a single website.
+
+For broad crawls, consider these adjustments:
+
+-   .. _broad-crawls-concurrency:
+
+    Increase the global concurrency:
+
+    -   Set :setting:`CONCURRENT_REQUESTS` as close to
+        :setting:`CONCURRENT_REQUESTS_PER_DOMAIN` × [number of target domains]
+        (e.g. 8 × 10 domains = 80 concurrent requests) as your CPU and memory
+        allow.
+
+    -   Increase :setting:`SCRAPER_SLOT_MAX_ACTIVE_SIZE` when increasing
+        :setting:`CONCURRENT_REQUESTS` stops making a difference.
+
+    If your CPU or memory become a bottleneck, see :ref:`optimize-resources`.
+
+-   Optimize request scheduling:
+
+    -   .. _broad-crawls-scheduler-priority-queue:
+
+        Set :setting:`SCHEDULER_PRIORITY_QUEUE` to
+        :class:`~scrapy.pqueues.DownloaderAwarePriorityQueue`.
+
+    -   .. _broad-crawls-bfo:
+
+        If memory is a bottleneck, see if :ref:`crawling in breadth-first order
+        (BFO) <faq-bfo-dfo>` lowers memory usage.
```

Comment:
The old docs said:

> In broad crawls, however, page crawling tends to be faster than page processing. As a result, unprocessed early requests stay in memory until the final depth is reached, which can significantly increase memory usage.

I am not sure how BFO addresses this, to be honest, that’s why I changed the wording to a “maybe it saves memory”, even though I feel it could actually increase memory.